images/router/haproxy/pod.json

```diff
@@ -45,6 +45,10 @@
           {
             "name": "OPENSHIFT_INSECURE",
             "value": "${OPENSHIFT_INSECURE}"
+          },
+          {
+            "name": "DROP_SYN_DURING_RESTART",
+            "value": "0"
```

Comment:
I don't really want to enable this by default, because the people who reported it seemed to have other issues that may have not ended up needing this at all, and I'm leery of putting it in production.  So it's more here as in case people hit something and we can tell them how to edit the dc to enable it... I'd rather not change the tooling yet.

You or @smarterclayton could convince me otherwise.